packages/account/src/Components/forms/form-fields.jsx

```diff
@@ -21,15 +21,16 @@ export const DateOfBirthField = ({ name, portal_id, ...rest }) => (
     </Field>
 );
 
-export const FormInputField = ({ name, optional = false, warn, ...rest }) => (
+export const FormInputField = ({ name, optional = false, ...rest }) => (
     <Field name={name}>
         {({ field, form: { errors, touched } }) => (
             <Input
                 type='text'
                 required={!optional}
```

Comment:
I think the `optional` prop can be removed as well as its assignment to `required`. I am not sure but I remeber removing this before.

Reply:
Just kept it for the time being as if from anywhere optional is being passed, we'll refactor in the upcoming form refactoring card